docs/apache-airflow-providers-apprise/connections.rst

```diff
@@ -60,6 +60,17 @@ Config (required)
         },
       ]
 
-    .. seealso::
-      - `Apprise URL Basics <https://github.com/caronc/apprise/wiki/URLBasics#apprise-url-basics>`_
-      - `Tagging <https://github.com/caronc/apprise/wiki/Development_API#tagging>`_
+
+Using Environment Variable
+''''''''''''''''''''''''''
+
+The configuration can be provided in an environment variable.
+
+.. code-block:: bash
+
+    AIRFLOW_CONN_APPRISE_DEFAULT='{"extra": {"config": {"path": "https://hooks.slack.com/services/T1JJ3T3L2/A1BRTD4JD/TIiajkdnlazkcOXrIdevi7F", "tags": "alert"}}}'
```

Comment:
Just wondering is this Slack Webhook URI related to any real URI?
Maybe better replace it to something which definitely looks like a fake URI, e.g.: 

`https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX`